pkg/utils/ioutil.go

```diff
@@ -120,3 +120,13 @@ func Copy(src, dst string) error {
 	}
 	return out.Close()
 }
+
+// Move moves a file from src to dst, this is done by copying the file and then
+// delete the old one. Use os.Rename() to rename file within the same filesystem
+// instead this, it's more lightweight but can not be used accross devices.
+func Move(src, dst string) error {
+	if err := Copy(src, dst); err != nil {
```

Comment:
Could we check if src and dst can be renamed and do that if possible, the copy and delete if not? Doesn't need to be done now, can be later.

Reply:
Yes it's possible to try rename first and check for the error, I'll see if there's a simple way of doing so.